docs/blog/2022-03-28-storswift-joining.md

```diff
@@ -0,0 +1,32 @@
+---
+title: StorSwift joins the League of Entropy
+summary: "StorSwift becomes the newest member of the League of Entropy. We've asked them why they're motivated by drand and decided to contribute - check what they said!"
+date: 2021-09-14
+---
+_"StorSwift becomes the newest member of the League of Entropy. We've asked them why they're motivated by drand and decided to contribute - check what they said!"_
+
+We are excited to announce that [StorSwift](https://storswift.com/), a technology company based in China, which provides cloud computing and storage services, is joining forces with the [League of Entropy](https://leagueofentropy.com/) (LoE)! StorSwift is becoming the 15th member to join the LoE and the first partner joining from China - a significant milestone for the LoE!
+
+We?ve asked StorSwift some key questions that we?ve previously asked other LoE members - see [here](https://drand.love/blog/2021/09/14/the-value-of-drand/) and [here](https://drand.love/blog/2021/10/28/the-value-of-drand-continued/) - on the value of drand. Here are the answers.
+
+---
+
+**Why is drand important as a protocol for the next generation of the Internet in your opinion?**
+
+The drand protocol elegantly provides a truly decentralized robust solution to the random numbers quality problem, which indeed is fundamental for secure computer systems.
+
+---
+
+**Why did you decide to join the League of Entropy? What value do you see in drand that is important for your mission (what convinced you to join)?**
```

Comment:
IMO this does not really answer the question, but I guess that it's what they answered.